test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test

```diff
@@ -211,7 +214,7 @@ Date: REGEX(.*)
   <contextAttributeList>
     <contextAttribute>
       <name>A1</name>
-      <type/>
+      <type>Room</type>
```

Comment:
Yes, definitely
Already covered in CHANGES_NEXT_RELEASE with this line:
- Fix: A few more of the 'responses without entityId::type filled in' found and fixed (Issue #585)